src/applications/facility-locator/components/ServiceTypeAhead.jsx

```diff
@@ -103,38 +143,50 @@ class ServiceTypeAhead extends Component {
             {showError && (
               <span className="usa-input-error-message" role="alert">
                 <span className="sr-only">Error</span>
-                Please choose a service type.
+                Please search for an available service.
               </span>
             )}
             <span id="service-typeahead">
               <input
                 {...getInputProps({
                   placeholder: 'like Chiropractor or Optometrist',
+                  onFocus: () => this.setState({ isFocused: true }),
+                  onBlur: e => {
+                    this.setState({ isFocused: false });
+                    this.props.onBlur(e);
+                  },
                 })}
                 id="service-type-ahead-input"
-                onBlur={this.props.onBlur}
               />
-              {isOpen && inputValue && inputValue.length >= 2 ? (
-                <div className="dropdown" role="listbox">
-                  {services
-                    .filter(specialty => this.shouldShow(inputValue, specialty))
-                    .map((specialty, index) => (
-                      <div
-                        key={this.getSpecialtyName(specialty)}
-                        {...getItemProps({
-                          item: specialty,
-                          className: this.optionClasses(
-                            index === highlightedIndex,
-                          ),
-                          role: 'option',
-                          'aria-selected': index === highlightedIndex,
-                        })}
-                      >
-                        {this.getSpecialtyName(specialty)}
-                      </div>
-                    ))}
-                </div>
-              ) : null}
+
+              {this.renderSearchForAvailableServicePrompt(inputValue)}
+              {this.renderTryAnotherServicePrompt(inputValue)}
+
+              {isOpen &&
+                inputValue &&
+                inputValue.length >= 2 && (
+                  <div className="dropdown" role="listbox">
+                    {services
+                      .filter(specialty =>
+                        this.shouldShow(inputValue, specialty),
+                      )
+                      .map((specialty, index) => (
+                        <div
+                          key={this.getSpecialtyName(specialty)}
+                          {...getItemProps({
+                            item: specialty,
+                            className: this.optionClasses(
+                              index === highlightedIndex,
+                            ),
+                            role: 'option',
+                            'aria-selected': index === highlightedIndex,
+                          })}
+                        >
+                          {this.getSpecialtyName(specialty)}
+                        </div>
+                      ))}
+                  </div>
+                )}
```

Comment:
What do you think about moving the block that renders this <div> into a function, to improve readability/organization?

Reply:
good suggestion